source/slang-capture-replay/replay/file-processor.h

```diff
@@ -0,0 +1,30 @@
+#ifndef FILE_PROCESSOR_H
+#define FILE_PROCESSOR_H
+
+#include <string>
+#include "../../core/slang-stream.h"
+#include "../util/capture-format.h"
+#include "slang-decoder.h"
+
+namespace SlangCapture
+{
+    class FileProcessor
```

Comment:
The name is too generic, what do you mean by `process a file`?

Reply:
act